packages/at_persistence_secondary_server/test/at_notification_keystore_test.dart

```diff
@@ -136,6 +136,22 @@ void main() async {
       var expiredKeys = await keyStore.getExpiredKeys();
       expect(0, expiredKeys.length);
     });
+    test('test hive key exceeds max allowed chars', () async {
+      var keyStore = AtNotificationKeystore.getInstance();
+      var atNotification = (AtNotificationBuilder()
+            ..toAtSign = '@bob'
+            ..fromAtSign = '@alice'
+            ..id = '123')
+          .build();
+      var key =
+          'iujpsefqvdzmtqthrqbaxqszxokaiutvpnbcphcjvjghpdxzdwywfsaowruwafmcudeoarfhuncezjkwbdvprcbujeptisxkjtztxogqqrrnjpqrdsjmcrpmpusrkzaksdfleyzsuarjhsqvxwicxulzqjzcwwjaupxzoqfwenkfonwhxtmwamiyzqqoesnreknrzwxazvykbybafrlwgqsyreudprnakoioqiwoqiwqdebbdwbdywebwydbwrr@alice';
```

Comment:
For readability, instead of long string of letters unclear as to what length they are, can we instead create and use a function which returns a random string of whatever length is required

Reply:
Done